Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -838,16 +831,49 @@ module Kitos.Utility.KendoGrid {
                                     dataTextField: "text",
                                     dataValueField: "id",
                                     optionLabel: entry.title,
+                                    enabled: true,
                                     change: e => {
-                                        var selectedId = e.sender.value();
-                                        const newSelection = entry.dropDownConfiguration.availableOptions.filter(x => x.id === selectedId);
-                                        entry.dropDownConfiguration.selectedOptionChanged(newSelection.length > 0 ? newSelection[0] : null);
+                                        this.triggerSelectedOptionChanged(e, entry);
                                         this.saveGridOptions();
                                     }
                                 }
                             }
                         };
                         break;
+                    case KendoToolbarImplementation.CustomTemplateDropDownList:
+                        
+                        toolbar.push({
+                            name: entry.id,
+                            text: entry.title,
```

Comment:
![image](https://user-images.githubusercontent.com/13116889/193782921-67a20587-f30c-43ca-a4bc-d1e7a9c626f0.png)

I think you need to be able to inject som css to change the default size.. so like the position which is a customizable parameter, so should standardwidth... then we can create some standard sizes in kitos.less and map to those classes